src/ShopPaymentIPNBundle/objects/db/TPkgShopPaymentIPN_TPkgShopPaymentHandlerGroup.class.php

```diff
@@ -55,9 +55,10 @@ public function isValidIP($sIP)
      * extending AbstractPkgShopPaymentIPNHandlerException.
      *
      * @param TPkgShopPaymentIPNRequest $oRequest
+     *
      * @trows AbstractPkgShopPaymentIPNHandlerException
      */
-    final public function handleIPN(TPkgShopPaymentIPNRequest $oRequest)
+    final public function handleIPN(TPkgShopPaymentIPNRequest $oRequest): void
```

Comment:
Why do we add a php return definition here instead of an annotation?

Reply:
Oh, this was an oversight - I corrected it. I checked the diff between this branch and 7.0.x and this was the only return type added.